docs/api.adoc

```diff
@@ -260,6 +260,36 @@ Import a new account from an external keyfile.
 <-- {"jsonrpc": "2.0", "result": "0x0047201aed0b69875b24b614dda0270bcd9f11cc", "id": 1}
 ----
 
+=== emerald_importHDAccount
```

Comment:
But, yeah, we can just inject handling for HD_Keyfile inside original callback
Thanks